Doc/library/exceptions.rst

```diff
@@ -38,15 +38,16 @@ information on defining exceptions is available in the Python Tutorial under
 Exception context
 -----------------
 
-When raising (or re-raising) an exception in an :keyword:`except` or
-:keyword:`finally` clause
-:attr:`__context__` is automatically set to the last exception caught; if the
-new exception is not handled the traceback that is eventually displayed will
+When raising a new exception and an exception
+is already being handled, the new exception's
+:attr:`__context__` attribute is automatically set to the originating
+exception.  An exception may be handled when an :keyword:`except` or
+:keyword:`finally` clause, or a :keyword:`with` statement, is used.  If the
+new exception is not handled, the traceback that is eventually displayed may
 include the originating exception(s) and the final exception.
```

Comment:
```suggestion
include the context exception(s) as well as the final exception.
```

Reply:
I wonder if the last sentence should be removed altogether because there is a paragraph below about the traceback.